modules/identity/src/Volo.Abp.Identity.HttpApi/Volo/Abp/Identity/IdentityUserLookupController.cs

```diff
@@ -9,7 +9,7 @@ namespace Volo.Abp.Identity
     [RemoteService]
     [Area("identity")]
     [ControllerName("UserLookup")]
-    [Route("api/identity/user-lookup")]
+    [Route("api/identity/user-lookups")]
```

Comment:
``api/identity/users/lookup`` would be better. Lookups is not so natural. Endpoints become:

* ``api/identity/users/lookup/{id}``
* ``api/identity/users/lookup/by-username/{userName}``

What do you think?